setup.sh

```diff
@@ -61,7 +61,7 @@ setup () {
 ## make targets
 BIN="bpkg"
 if [ -z "$PREFIX" ]; then
-  if [ $USER == 'root' ]; then
+  if [[ $USER == "root" ]]; then
```

Comment:
Shouldn't this have double quotes for `$USER` (as it should be parsed) and single quotes for `root` (as it need not be parsed)?

i.e:
```
 if [[ "$USER" == 'root' ]]; then
```

Reply:
yes, you're right! Making changes now